.vsts-dotnet.yml

```diff
@@ -23,9 +23,12 @@ variables:
   - name: SourceBranch
     value: $(IbcSourceBranchName)
   # If we're not on a vs* branch, use main as our optprof collection branch
+  # NOTE: the code is temporarily fixed. For the branches that should use opt-prof from the main branch we should use the latest working Opt-Prof 20220901.6-001 collected from main 20220901.6.
   - ${{ if not(startsWith(variables['Build.SourceBranch'], 'refs/heads/vs')) }}:
+    - name: OptProfDrop
```

Comment:
I think the next section overwrites this? It checks if it's the string 'default' and gives it a new value if so...

Reply:
It overwrites this, as it should be, because that is the case of non-default `OptProfDropName`. It was manually set and this setting should be respected. Our point is to overwrite the default value.